2021-offseason/src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,19 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class kHopper{
+        public static final int LEFT_ID = 0;
+        public static final int RIGHT_ID = 0;
+        public static final int KICKER_ID = 0;
```

Comment:
I'm assuming these will be changed once you have the real IDs?

Reply:
Yeah we just don't know which motor controllers will be actually used on the robot right now